pkg/api/aim/experiments.go

```diff
@@ -327,3 +335,77 @@ func DeleteExperiment(c *fiber.Ctx) error {
 		"status": "OK",
 	})
 }
+
+func UpdateExperiment(c *fiber.Ctx) error {
```

Comment:
I know we're going to refactor this soon, but rather than put this net new method here where it soon will be moved (and without unit tests), can we instead create the first repository object for aim and put it there (with tests)? @jgiannuzzi  - what do you think?

Reply:
I think there is no big difference right now. My vote to have it here and keep everything similar(even if it looks bad) and then in scope of separate tasks make it better and well structured.

Reply:
so yep, looks a bit ugly and overloaded but I hope we an normalise it very soon, because otherwise in scope of this task we will have even more changes we need which we always try to avoid.

Reply:
okay